arangod/Agency/Agent.cpp

```diff
@@ -548,10 +552,9 @@ void Agent::sendAppendEntriesRPC() {
         arangodb::rest::RequestType::POST, path.str(),
         std::make_shared<std::string>(builder.toJson()), headerFields,
         std::make_shared<AgentCallback>(this, followerId, highest, toLog),
-        3600.0, true);
-      // Note the timeout is essentially indefinite. We let TCP/IP work its
-      // magic here, because all we could do would be to resend the same 
-      // message if a timeout occurs.
+        150.0, true);
```

Comment:
_earliestPackage[followerId] was set to now() + 3600 just a few lines above.  Why is this change to 150.0 ok without a similar change to the 3600 above it?

Reply:
Actually, the + 3600 above was changed to + 30. I think the 3600 was meant to say "never". The rationale now is that a retry is done after 30s, and the timeout is set to 150 to have at most 5 requests in flight concurrently. Since we only ever send batches of up to 99 transactions, this ought to be enough.